service/src/main/java/com/codeforcommunity/requester/Emailer.java

```diff
@@ -93,4 +93,16 @@ public void sendAccountDeactivatedEmail(String sendToEmail, String sendToName) {
     emailBody.ifPresent(
         s -> emailOperations.sendEmail(sendToName, sendToEmail, subjectAccountDeleted, s));
   }
+
+  public void sendInviteTeamEmail(String sendToEmail, String sendToName, String teamName) {
```

Comment:
Add a `// TODO` comment above this method so we don't forget to come back to it.

Reply:
Ticket in Monday